packages/metrics/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "@k8slens/typescript/config/base.json",
```

Comment:
Should this be in the dependencies or does it just work?

Reply:
hmm good question.. no other package imports this but all of them use it... it should probably be a dev dependency for all?

Reply:
added